openupgrade_scripts/scripts/hr_timesheet/17.0.1.0/post-migration.py

```diff
@@ -0,0 +1,25 @@
+from openupgradelib import openupgrade
+
+
+def _project_update_fill_timesheet(env):
+    updates = env["project.update"].with_context(active_test=False).search([])
+    for update in updates:
+        project = update.project_id
+        company = project.company_id
```

Comment:
@duong77476-viindoo nếu project.company_id.timesheet_encode_uom_id không có thì không cần chạy xuống dưới
dựa vào `env.company` sẽ không chuẩn, env lúc này là env lúc chạy mig, còn env của 17 là env của người dùng
```suggestion
        company = project.company_id
```